### ISSUE TITLE
IS-07: exception not caught in case bool value is sent instead of string

### ISSUE THREAD

**Issue by maweit** [AMWA-TV/nmos-testing#436]
test_06_02: if for a string type a bool value is provided as payload an exception is not caught: 
Uncaught exception. Please report the traceback from the terminal to https://github.com/amwa-tv/nmos-testing/issues. object of type 'bool' has no len()

**Comment by garethsb-sony** [comment]
Yes, similar to #435. Schema validation actually needs to be applied or the type checked or that exception caught.

**Comment by maweit** [comment]
The following line would (at least) catch the exception - shell i create a PR? 
```
except TypeError as e:
            return test.FAIL("Source {} JSON type error: {}".format(source_id, e))
```

**Comment by garethsb-sony** [comment]
Yes, please.

**Comment by garethsb-sony** [comment]
Resolved by #437.
